test/screenshot/infra/lib/github-api.js

```diff
@@ -43,14 +43,16 @@ class GitHubApi {
 
   constructor() {
     this.gitRepo_ = new GitRepo();
-    this.octokit_ = octokit();
+    this.octokit_ = octokit(this.getAuthToken_());
```

Comment:
Why are there screenshot testing infra changes in this PR?

Reply:
I regenerated `package-lock.json`, and evidently some of our dependencies don't follow semver, so we got a few breaking changes. I figured I might as well fix them while I'm here.